jinahub/encoders/image/CLIPImageEncoder/README.md

```diff
@@ -1,54 +1,68 @@
 # CLIPImageEncoder
 
-**CLIPImageEncoder** is an image encoder that wraps the image embedding functionality using the **CLIP** model from huggingface transformers.
+**CLIPImageEncoder** is an image encoder that wraps the image embedding functionality using the [CLIP](https://huggingface.co/transformers/model_doc/clip.html) model from huggingface transformers.
 
-It takes `Document`s with images stored in the `blob` attribute as inputs, and stores the
-resulting image embedding in the `embedding` attribute. You can store original images in
-the `blob` attribute, and they should be in the RGB format and have a shape `[H, W, 3]`). You
-can also choose to pass in already pre-processed images (see the class documentation).
+This encoder is meant to be used in conjunction with the [CLIPTextEncoder](https://github.com/jina-ai/executors/tree/main/jinahub/encoders/text/CLIPTextEncoder),
+as it can embedd text and images to the same latent space.
 
-The **CLIP** model was originally proposed in [Learning Transferable Visual Models From Natural Language Supervision](https://arxiv.org/abs/2103.00020), and is trained to embed images and text to the same latent
-space. The corresponding text encoder is **[CLIPTextEncoder](https://hub.jina.ai/executor/livtkbkg)**,
-using both encoders together works well in multi-modal or cross-modal search applications.
+The following arguments can be passed on initialization:
+
+- `pretrained_model_name_or_path`: name or path of the pre-trained CLIP model.

```

Comment:
We don't want the arguments in the README, right? They are parsed by Hubble and displayed in the hub.jina.ai interface